### ISSUE TITLE
Fix Portfolio Page Keyboard Accessibility 

### ISSUE THREAD

**Issue by vkoves** [vkoves/vkoves.github.io#12]
The portfolio page has icons that only show up on hover, like so:

![screenshot from 2018-12-09 16-21-40](https://user-images.githubusercontent.com/3187531/49703738-89e32680-fbce-11e8-8622-af33fe5e66bf.png)

Since these icons only show up on hover, a keyboard user cannot access them. This should be remedied, likely by adding the icons below each portfolio item, with them always visible.
